app/services/admin/onboarding_wizard.rb

```diff
@@ -136,7 +136,8 @@ def load_setup_steps(community_id)
 
     def init_setup_steps(community_id)
       community = Community.find(community_id)
-      community_customizations = CommunityCustomization.where(community_id: community.id)
+      locales = community.locales
+      community_customizations = CommunityCustomization.where(community_id: community.id).select { |cc| locales.include?(cc.locale) }
```

Comment:
Does this change apply to the problems that have been encountered with other steps of the onboarding wizard, like custom fields.